packages/dart/noports_core/lib/src/srvd/signature_verifying_socket_authenticator.dart

```diff
@@ -66,55 +66,78 @@ class SignatureAuthVerifier {
     Completer<(bool, Stream<Uint8List>?)> completer = Completer();
     bool authenticated = false;
     StreamController<Uint8List> sc = StreamController();
-    logger.info('SignatureAuthVerifier $tag: starting listen');
+    logger.info('SignatureAuthVerifier for $tag: starting listen');
+    List<int> buffer = [];
     socket.listen((Uint8List data) {
       if (authenticated) {
         sc.add(data);
       } else {
```

Comment:
The changes below here are all about ensuring that the SignatureAuthVerifier is resilient to situations where the authentication data comes in separate flows, or comes in the same flow as the first part of the ssh handshake